### PR TITLE
fix(bag): review asset buttons

### PR DESCRIPTION
**Pull request by JFrankfurt**
https://uniswaplabs.atlassian.net/browse/WEB-2015

### CODE REVIEW

File: src/components/Button/index.tsx [hunk 1]
```diff
@@ -366,33 +366,121 @@ export function ButtonRadioChecked({ active = false, children, ...rest }: { acti
...
+  :hover {
+    ${ButtonOverlay} {
+      background-color: ${({ theme }) => theme.stateOverlayHover};
+    }
   }
 `
+
+interface ThemeButtonProps extends React.ComponentPropsWithoutRef<'button'>, BaseButtonProps {}
+
+export const ThemeButton = ({ ...rest }: ThemeButtonProps) => {
```
Comment (aballerr) [inline src/components/Button/index.tsx:478]: Should it be this?

```{children, ...rest}```

Comment (JFrankfurt) [reply src/components/Button/index.tsx:478]: yes ty

Comment (JFrankfurt) [reply src/components/Button/index.tsx]: done

File: src/nft/components/bag/BagRow.tsx [hunk 2]
```diff
@@ -24,13 +24,20 @@ import styled from 'styled-components/macro'
 
 import * as styles from './BagRow.css'
 
-const RemoveButton = styled(MediumButton)`
+const RemoveButton = styled(ThemeButton)`
   border-radius: 12px;
```
Comment (vm) [inline src/nft/components/bag/BagRow.tsx:28]: ideally shouldn't need some of this theming now?

**Review by vm — APPROVED** [review]
would ideally live in nft/components instead of top-level, but i'm okay with this incremental step towards a shared button

**Pull request merged**